### PR TITLE
Seperate out intro cdrom command for Windows and other OSes.  Also add example for mounting cue files.

### PR DESCRIPTION
**Pull request by bmunger**

### CODE REVIEW

**Comment by kcgen** [comment]
@bmunger, looks great - thank you!

File: src/dos/dos_programs.cpp [hunk 1]
```diff
@@ -1179,7 +1179,11 @@ class INTRO : public Program {
 		/* Only run if called from the first shell (Xcom TFTD runs any intro file in the path) */
 		if (DOS_PSP(dos.psp()).GetParent() != DOS_PSP(DOS_PSP(dos.psp()).GetParent()).GetParent()) return;
 		if (cmd->FindExist("cdrom",false)) {
-			WriteOut(MSG_Get("PROGRAM_INTRO_CDROM"));
+#if (WIN32)
```
Comment (dreamer) [inline src/dos/dos_programs.cpp:1182]: This should be either
```C
#ifdef WIN32
```
or
```C
#if defined(WIN32)
```

**Review by dreamer — CHANGES_REQUESTED** [review]
Looks good - initially I was concerned about ifdef around translation string, but the way it is written right now it will work correctly :)

File: src/dos/dos_programs.cpp (hunk 1 above)
Comment (bmunger) [reply src/dos/dos_programs.cpp]: Thanks!  I thought it was unusual to see it handled this way too, but I was keeping with the convention used there.  I made a commit to address that.

**Comment by kcgen** [comment]
Looks good; thanks @bmunger!
Can you squash the two commits into once, and force-push?  Then I will merge.

**Comment by bmunger** [comment]
I initially made a second commit so it wouldn't invalidate the change request.  I have squashed the second commit into the first and forced pushed.  Should be good now.  Thanks!

**Review by kcgen — APPROVED** [review]

**Pull request merged**